doc/design/mvp.md

```diff
@@ -0,0 +1,85 @@
+# Kube Arbitrator Roadmap/MVP
+
+@k82cn, @foxish, 08/27/2017
+
+
+## Overview
+
+[Resource sharing architecture for batch and serving workloads in Kubernetes](https://docs.google.com/document/d/1-H2hnZap7gQivcSU-9j4ZrJ8wE_WwcfOkTeAGjzUyLA/edit#heading=h.a1k69dgabg0w) proposed `QueueJob` feature to run batch job with services workload in Kuberentes. Considering the complexity, the whole batch job proposal was separated into two phase: `Queue` and `QueueJob`. Refer to the following roadmap for the detail of `Queue` and `QueueJob`
+
+## infra (MVP)
+
+### code generated
+
+1. Generate Deepcopy: **Done**
+
+### test-infra
+
+kube-arbitrator focus on policy based resource sharing, so unit test and integration test is enough. 
```

Comment:
@k82cn - From what i read, this won't make usage `kubernetes/test-infra`, so no e2e tests will be provided ?

Reply:
Will not provide e2e tests in MVP, as we focus on policy, almost no feature interaction with other components. In the future, e2e is required.

Reply:
okay, later do you intend to use same infra. as for `kubernetes/kubernetes` ?

Reply:
hm... not sure now, it dependent on how complex it is. If someone help on it, that'll be great. But integration test framework is required.